tox.ini

```diff
@@ -4,65 +4,42 @@ isolated_build = true
 
-[testenv:deps]
-description = Update dependencies by running pip-compile-multi
-skip_install = true
-deps = pip-compile-multi
-changedir = requirements
-commands = pip-compile-multi -d .
+[testenv:docs]
+description = invoke sphinx-build to build the HTML docs
+deps =
+  {posargs:}
+  -r requirements/docs.txt
+allowlist_externals=find
+setenv =
+  {[testenv]setenv}
+  # Search directory for confuse for custom config enabling plopp plotting
+  SCIPPDIR={toxinidir}{/}docs{/}buildconfig
+commands = python -m sphinx -j2 -v -b html -d {toxworkdir}/docs_doctrees docs html
+           python -m sphinx -j2 -v -b doctest -d {toxworkdir}/docs_doctrees docs html
+           find html -type f -name "*.ipynb" -not -path "html/_sources/*" -delete
 
-[testenv:config-build]
-description = Build new static default configuration yaml file of BeamlimeSystem and export it into the generated resource directory
-changedir = src/beamlime/resources
-allowlist_externals = python
-commands =
-    - python -m beamlime.executables.export_preset_configs --directory {toxinidir}/src/beamlime/resources/generated/ --overwrite
+[testenv:linkcheck]
+description = Run Sphinx linkcheck
+deps = -r requirements/docs.txt
+commands = python -m sphinx -j2 -v -b linkcheck -d {toxworkdir}/docs_doctrees docs html
 
 [testenv:static]
 description = Code formatting and static analysis
+skip_install = true
 deps = -r requirements/static.txt
 allowlist_externals = sh
 # The first run of pre-commit may reformat files. If this happens, it returns 1 but this
 # should not fail the job. So just run again if it fails. A second failure means that
 # either the different formatters can't agree on a format or that static analysis failed.
 commands = sh -c 'pre-commit run -a || (echo "" && pre-commit run -a)'
 
-[testenv:docs]
-deps = -r requirements/docs.txt
-description = Build documentation
-allowlist_externals = find
-commands =
-  python -m sphinx -v -b html -d doctrees docs html
-  python -m sphinx -v -b doctest -d doctrees docs html
-  find html -type f -name "*.ipynb" -not -path "html/_sources/*" -delete
-
-[testenv:docs-manual]
-deps = -r requirements/docs-manual.txt
-description = Build diagram manually
-allowlist_externals =
-    python
-    mv
-    find
-commands =
-  - python {toxinidir}/docs/resources/diagram.py
-  - manim -qh -t docs/resources/draw_logo.py CreateLogoImage -s --media_dir docs/resources/manim/ -o logo.png -r 1024,512
-  - mv docs/resources/manim/images/draw_logo/logo.png docs/resources/logo.png
-  - manim -qh -t docs/resources/draw_logo.py CreateIconImage -s --media_dir docs/resources/manim/ -o icon.png -r 256,256
-  - mv docs/resources/manim/images/draw_logo/icon.png docs/resources/icon.png
-  - manim -qh docs/resources/draw_logo.py CreateLogoVideo --media_dir docs/resources/manim/ -o logo
-  - mv docs/resources/manim/videos/draw_logo/1080p60/logo.mp4 docs/resources/
-  - find docs/resources/manim -delete
```

Comment:
Removed intentionally?

Reply:
Yes, I'll add a similar one later in the branch that I'm working on the documentation.